web/template/add-contact.gotmpl

```diff
@@ -0,0 +1,157 @@
+{{ template "page" . }}
+{{ define "main" }}
+{{ template "content-header" . }}
+{{ template "error-summary" (rename_errors .Errors) }}
+    <header>
+        <h1 class="govuk-heading-l  govuk-!-margin-bottom-1">
+            Create new contact
+        </h1>
+    </header>
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-one-half">
+            <form id="add-contact-form" class="" method="post">
+                <input
+                    type="hidden"
+                    name="xsrfToken"
+                    value="{{ .XSRFToken }}" />
+                <div class="govuk-form-group">
+                    <label class="govuk-label" for="deputy">Name (required)</label>
```

Comment:
All these labels have the same `for`. Labels should be paired with the element they are labelling using `for` and `id`.